src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,23 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <div className="card" key={movie.imdbId}>
+        <MovieCard {...movie} />
+      </div>
+    ))}
```

Comment:
```suggestion
    {movies.map(movie => (
      <MovieCard {...movie} key={movie.imdbId} />
    ))}
```
Этот див лучше перенести в компонент Card